README.md

```diff
@@ -0,0 +1,103 @@
+o-sliding-popup [![Circle CI](https://circleci.com/gh/Financial-Times/o-sliding-popup/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-sliding-popup/tree/master)
+=================
+
+Next-UI Sliding Popup.
```

Comment:
Initially developed this as an `o-` component. Shall I just change this to `n-`? Or is there more to do?